src/components/game/components/board/Board.js

```diff
@@ -59,31 +58,32 @@ const Board = ({
       </div>
 
       <div className="buttons-row">
-        <div className="board-row">
-          <h2 className="text">Money: {playerMoney}</h2>
-          <h2 className="text">Bid: {playerBet}</h2>
-          <h2 className="text">
-            Strength: {phase >= 1 ? handCheckToMsg(playerHand) : null}
-          </h2>
-        </div>
+        <Stats
+          money={playerMoney}
+          bid={playerBet}
+          strength={phase >= 1 ? handCheckToMsg(playerHand) : null}
+          visible={true}
```

Comment:
visibility is a parents concern as said on the previous review.

Reply:
Done.